Content.Server/Chemistry/EntitySystems/HypospraySystem.cs

```diff
@@ -44,25 +44,29 @@ private void UseHypospray(Entity<HyposprayComponent> entity, EntityUid target, E
             TryDraw(entity, target, drawableSolution.Value, user);
         }
 
-        TryDoInject(entity, target, user);
+        return TryDoInject(entity, target, user);
     }
 
     private void OnUseInHand(Entity<HyposprayComponent> entity, ref UseInHandEvent args)
     {
         if (args.Handled)
             return;
 
-        TryDoInject(entity, args.User, args.User);
-        args.Handled = true;
+        if (TryDoInject(entity, args.User, args.User))
+        {
+            args.Handled = true;
+        }
```

Comment:
This doesn't make a difference since earlier code for all of these already checks if it was handled. Is this a coding standard requirement? I recall a lot of other handlers usually use the form I'm currently using. If that is outdated, then yeah I can fix that.

Reply:
its just a shorter way of doing the same, it's the preferred pattern for this kind of thing